examples/Basic/package.json

```diff
@@ -6,7 +6,8 @@
     "start": "node_modules/react-native/packager/packager.sh"
   },
   "dependencies": {
-    "react-native": "^0.11.4",
-    "react-native-blur": "^0.7.0"
+    "react": "^0.14.5",
+    "react-native": "^0.23.0",
+    "react-native-blur": "github:cmcewen/react-native-blur"
```

Comment:
I would actually suggest doing the same we did in side-menu, so that we can dynamically reference `master` w/o going through npm. Can be done in a separate PR.